<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientDialog</class>
 <widget class="QDialog" name="PatientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>病人信息</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_name">
       <property name="text">
        <string>姓名*:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_gender">
       <property name="text">
        <string>性别:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="genderCombo"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_birth">
       <property name="text">
        <string>出生日期:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDateEdit" name="birthDateEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_phone">
       <property name="text">
        <string>电话:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="phoneEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_email">
       <property name="text">
        <string>邮箱:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="emailEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_address">
       <property name="text">
        <string>地址:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="addressEdit"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_idcard">
       <property name="text">
        <string>身份证号:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="idCardEdit"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_history">
       <property name="text">
        <string>病史:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QTextEdit" name="medicalHistoryEdit">
       <property name="maximumHeight">
        <number>80</number>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_allergy">
       <property name="text">
        <string>过敏信息:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QTextEdit" name="allergyInfoEdit">
       <property name="maximumHeight">
        <number>80</number>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_emergency">
       <property name="text">
        <string>紧急联系人:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="emergencyContactEdit"/>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_emergency_phone">
       <property name="text">
        <string>紧急联系电话:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="emergencyPhoneEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PatientDialog</receiver>
   <slot>accept()</slot>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PatientDialog</receiver>
   <slot>reject()</slot>
  </connection>
 </connections>
</ui>
